src/app/frontend/replicasetlist/replicasetcard.scss

```diff
@@ -19,34 +19,54 @@
   width: ($layout-breakpoint-lg - (2 * $baseline-grid) * 2) / 3;  
 }
 
-// This override button style to make it look like a link.
-.kd-replicaset-card-logs {
-  border-radius: inherit;
+.kd-replicaset-card-name {
   color: inherit;
-  font-size: inherit;
-  font-weight: inherit;
-  line-height: 0;
-  margin: inherit;
-  min-height: inherit;
-  min-width: inherit;
-  padding: inherit;
+  display: inline-block;
+  font-weight: $title-font-weight;
   text-decoration: none;
-  text-transform: inherit;
+  width: 100%;
```

Comment:
I'm testing the UI and I've noticed that when name is short, clickable space to enter details page is very wide(it's always flexed). It doesn't feel very intuitive when I'm clicking on blank spot and I'm redirected to details page. Either we can make whole card area clickable or adjust card name width. It could be also more noticable that name can be clicked to perform action.